xc-admin/packages/xc-admin-common/src/governance_payload/index.ts

```diff
@@ -0,0 +1,105 @@
+import { ChainId, ChainName, toChainName } from "@certusone/wormhole-sdk";
+import * as BufferLayout from "@solana/buffer-layout";
+
+export declare const Action: {
+  readonly ExecutePostedVaa: 0;
+  readonly UpgradeContract: 0;
```

Comment:
it's surprising that there are two 0s here. (I understand why it's the case, but it will confuse the next person who reads this.) Either document why there are two 0s and why its ok, or consider making different versions of Action for each module, or define this in a way where each Action corresponds to a tuple of the moduleId and actionId